### PR TITLE
Create an initial content of the README.md file

### PR DESCRIPTION
**Pull request by stefaniuk**
Fixes nhs-england-tools/repository-template#1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,60 @@
...
+  - [Architecture](#architecture)
+    - [Diagrams](#diagrams)
+    - [Configuration](#configuration)
+  - [Contributing](#contributing)
+  - [Contacts](#contacts)
+  - [Licence](#licence)
+
+## Installation
+
+By including preferably a one-liner or if necessary a set of clear CLI instructions we increase user experience. This should be a frictionless installation process that works on various operating systems (macOS, Linux, Windows WSL) and handles all the dependencies.
```
Comment (chrisw-dev) [inline README.md:22]: Should this read `we improve user experience` ?

**Review by chrisw-dev — COMMENTED** [review]
One wording change suggestion

**Review by chrisw-dev — APPROVED** [review]
LGTM

**Review by JosephKelly — APPROVED** [review]

**Review by walteck — APPROVED** [review]
LGTM

**Pull request merged**